state-sync/diem-data-client/src/diemnet/tests.rs

```diff
@@ -381,7 +382,8 @@ async fn bad_peer_is_eventually_added_back() {
 
     // Advance time so the poller sends a data summary request.
     tokio::task::yield_now().await;
-    mock_time.advance_async(DATA_SUMMARY_POLL_INTERVAL).await;
+    let data_summary_poll_interval = DiemDataClientConfig::default().data_summary_poll_interval;
+    mock_time.advance_async(data_summary_poll_interval).await;
```

Comment:
nit: inline `data_summary_poll_interval` maybe?

Reply:
Made the poll interval constant: `const SUMMARY_POLL_INTERVAL: Duration = Duration::from_millis(1_000)`